### ISSUE TITLE
LibBulletDynamics.so.2.83 shipped instead of 2.87

### ISSUE THREAD

**Issue by Derpford** [janusvr/janus#38]
Janus will not launch, complaining of a missing library:
```./janusvr: error while loading shared libraries: libBulletDynamics.so.2.87: cannot open shared object file: No such file or directory```
However, an older version of this library is shipped with JVR on the download page:
```
> ls | grep libBullet
libBulletCollision.so.2.83
libBulletDynamics.so.2.83
```
This is somewhat frustrating.

**Comment by JamesMcCrae** [comment]
Thank you for reporting this.

Commit https://github.com/janusvr/janus/commit/520187a6f36def9aace8cbeeada174ddb7bb926d updates those libraries to 2.87.  You can check the latest build status at http://builds.janusvr.com and the builds hosted there are linked from our main website.

Please let me know if you still have problems running Janus on Linux (because of this, or any other outdated library that is still being packaged).

**Comment by JamesMcCrae** [comment]
Just did two more commits to address your issues:

https://github.com/janusvr/janus/commit/7a2a85777289187067c183fd5c02b17d529fd22e

https://github.com/janusvr/janus/commit/bf5804fe5d264205708e62e14fe2d09ff3e5e2dc

Continue to inform if you still have issues compiling.

**Comment by Derpford** [comment]
`libQt5Xml.so.5`, `libQt5Script.so.5`, and `glibc` version 2.27 are all missing. `glibc` is only available on Leap 15 up to version 2.26.
